context/app/static/js/components/Detail/visualization/VisualizationNotebookButton/VisualizationNotebookButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import GetAppRoundedIcon from '@material-ui/icons/GetAppRounded';
+import styled from 'styled-components';
+
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+import { WhiteBackgroundIconButton } from 'js/shared-styles/buttons';
+
+// TODO: Would it be ok to create visualization/style.js and put this there,
+// so all the buttons would use it?
+// Individual buttons padding on the left and right seems fragile, if they get rearranged.
+const StyledSecondaryBackgroundTooltip = styled(SecondaryBackgroundTooltip)`
```

Comment:
Sure, creating a visualization/style.js works. Should the margin be on the button and not the tooltip?